en/docs/sdks/start-integrating-apps/integrate-a-react-app.md

```diff
@@ -54,6 +54,20 @@ To add CORS configurations:
     !!! note
         Make a note of the **OAuth Client Key** and **OAuth Client Secret** that appear, as they will be used to configure the sample application.
 
+## Configure the Resident Identity Provider
+
+!!! Note
+    Follow this step, only if you are running the WSO2 Identity Server on a custom domain/port.
```

Comment:
Shall we add this as an expandable?

??? Note "If you are running WSO2 Identity Server on a custom domain"

Reply:
Removed the info box and kept it as normal text. WDYT?